### ISSUE TITLE
totalLockedValue [sic] is a fantasy number

### ISSUE THREAD

**Issue by kremalicious** [oceanprotocol/ocean-subgraph#16]
[This query](https://subgraph.mainnet.oceanprotocol.com/subgraphs/name/oceanprotocol/ocean-subgraph/graphql?query=%7B%0A%20%20poolFactories%20%7B%0A%20%20%20%20totalLockedValue%0A%20%20%20%20totalLiquidity%0A%20%20%20%20totalSwapVolume%0A%20%20%20%20totalSwapFee%0A%20%20%20%20poolCount%0A%20%20%20%20finalizedPoolCount%0A%20%20%7D%0A%7D%0A) returns as `totalLockedValue`:

```
OCEAN 608499837
```

which would be converted:

```
€ 287,748,011
```

We do not have 287 Million Euro locked in our pools. So it's completely wrong.

---
(And it should be `totalValueLocked` not `totalLockedValue`. Already commented in the [PR](https://github.com/oceanprotocol/market/pull/353/files))

**Comment by trentmc** [comment]
As a matter of process: why is this a new issue, rather than a comment in the [issue](https://github.com/oceanprotocol/market/issues/286) & [PR](https://github.com/oceanprotocol/market/pull/353/files) for bringing in TVL?

Also note: it should be "TVL" not "TLV". I've commented in the PR, but I'm commenting here to help ensure that the mistake doesn't propagate:)

**Comment by kremalicious** [comment]
because this is the place where it needs to be solved so it's an issue here. Issues should exists where they are solved. And the PR https://github.com/oceanprotocol/market/pull/353 then states in its description that it is blocked by this. `totalLockedValue` is how it is called in multiple places here in this repo in the code, it's the current reality. So that would be an new issue in this repo to make sure to change all those keys.

**Comment by mihaisc** [comment]
As soon as we merge both PRs (#6 and #17, they are prio for me now ) i'll take a look at this

**Comment by kremalicious** [comment]
so the fix for this was combined in #6 and not in separate PR? Did the rename of the key happen too?

**Comment by mihaisc** [comment]
It includes the rename but it was not deployed yet, will check and close the issue.

**Comment by kremalicious** [comment]
yup, know the full deployments can't be that easily automated but we should still do a release with a changelog right before doing deployment. Then it's clear to users which version of code is deployed, and what changed. Like, renaming that key is a breaking change, but there's not even a commit message for it. Using PRs and `npm run release` will do that
